### PR TITLE
[UIExplorer] Android - Fixing #4392

### PR DESCRIPTION
**Pull request by mihaigiurgeanu**
This is just an workaround for making UIExplorer work in Android.

### CODE REVIEW

**Comment by janhenkgerritsen** [comment]
I ran into the same issue when trying to run the movies example. This pull request fixes the issue for me.

File: Libraries/Components/MapView/MapView.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ var EdgeInsetsPropType = require('EdgeInsetsPropType');
 var Image = require('Image');
 var NativeMethodsMixin = require('NativeMethodsMixin');
 var Platform = require('Platform');
-var RCTMapConstants = require('NativeModules').UIManager.RCTMap.Constants;
+var RCTMapConstants = require('NativeModules').UIManager.RCTMap?require('NativeModules').UIManager.RCTMap.Constants:undefined;
```
Comment (tadeuzagallo) [inline Libraries/Components/MapView/MapView.js]: Can you extract that?

```js
var RCTMap = require('NativeModules').UIManager.RCTMap;
var RCTMapConstants = RCTMap.Constants;
````

Comment (baoti) [inline Libraries/Components/MapView/MapView.js]: :+1: `var RCTMap = require('UIManager').RCTMap;`

Comment (nicklockwood) [inline Libraries/Components/MapView/MapView.js]: Yep, that's definitely the preferred approach :-)

**Comment by christopherdro** [comment]
+1
@mihaigiurgeanu Any chance you can make those revisions so we can get this merged in?

Thanks!

**Comment by christopherdro** [comment]
@mihaigiurgeanu Awesome! Any chance you can squash the changes down to 1 commit? Thanks

**Comment by mihaigiurgeanu** [comment]
Ok, I just did. Sorry for the delay, I don't know exactly how things work around here.

**Comment by tadeuzagallo** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**